### ISSUE TITLE
Migrations on deploy broken (regeneratorRuntime not found)

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#1085]
## Issue - Bug

I observed this during the last production deployment:

```
remote:        > node ./src/server/database/migrate.babel.js up --all        
remote:        [migrate-rethinkdb] Connecting to database        
remote:        Creating a pool connected to dokku-rethinkdb-rethinkdb:28015        
remote:        [migrate-rethinkdb] Connected        
remote:        [migrate-rethinkdb] Migration failed        
remote:        ReferenceError: regeneratorRuntime is not defined        
remote:            at /app/src/server/database/migrations/20170515134300-fixTaggedContent.js:1:1        
remote:            at Object.<anonymous> (/app/src/server/database/migrations/20170515134300-fixTaggedContent.js:1:1)        
remote:            at Module._compile (module.js:571:32)        
remote:            at loader (/app/node_modules/babel-register/lib/node.js:144:5)        
remote:            at Object.require.extensions.(anonymous function) [as .js] (/app/node_modules/babel-register/lib/node.js:154:7)        
remote:            at Module.load (module.js:488:32)        
remote:            at tryModuleLoad (module.js:447:12)        
remote:            at Function.Module._load (module.js:439:3)        
remote:            at Module.require (module.js:498:17)        
remote:            at require (internal/module.js:20:19)        
remote:            at /app/node_modules/migrate-rethinkdb/lib/migrate.js:201:16        
remote:            at Array.map (native)        
remote:            at /app/node_modules/migrate-rethinkdb/lib/migrate.js:198:37        
remote:            at lib$es6$promise$$internal$$tryCatch (/app/node_modules/ts-fs-promise/node_modules/es6-promise/dist/es6-promise.js:331:16)        
remote:            at lib$es6$promise$$internal$$invokeCallback (/app/node_modules/ts-fs-promise/node_modules/es6-promise/dist/es6-promise.js:343:17)        
remote:            at lib$es6$promise$$internal$$publish (/app/node_modules/ts-fs-promise/node_modules/es6-promise/dist/es6-promise.js:314:11)        
```

We should make sure this is patched up in the editor epic, if not sooner...

- **OS:** Linux

**Comment by mattkrick** [comment]
Ah yes, the ol await/async thing. I'll change out the babel when we're
ready to merge

**Comment by jordanh** [comment]
Fix in #1075
